### DISCUSSION TITLE
part-ii/2009-99/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#144] [Question comments]
Part II 2009 Number Theory: Paper 1, Section I, G 

https://questions.tripos.org/part-ii/2009-99/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

Theorem: (Chinese Remainder Theorem) Let $m_1, \dots, m_n$ be pairwise coprime positive integers and let $a_1, \dots, a_n \in \mathbb{Z}$. Then the system of congruences 

$$x \equiv a_i \pmod{i}, 1 \leq i \leq n$$

is soluble and the solution is unique $\mod M = \prod_{i=1}^{N} m_i$.

We use the Chinese Remainder Theorem to solve the second half of the question. First, we apply it to the first two congruences; we see by inspection that 2 is a solution, and by the Chinese Remainder Theorem the solution is therefore $x \equiv 2 \pmod{15}$. 

To incorporate the third congruence, it is possible to use inspection, but we can also use a different method involving Euclid's algorithm. We have that $15 - 2 \cdot 7 = 1$, so $15 \equiv 1 \pmod{7}$ and $15 \equiv 0 \pmod{15}$. Similarly, $-2 \cdot 7 \equiv -14 \equiv 1 \pmod{15}$ and $-14 \equiv 0 \pmod{7}$. Hence a solution is given by 

$$x = 2 \cdot (-14) + 6 \cdot 15 = -28 + 90 = 62$$

and by the Chinese Remainder Theorem, since $15 \cdot 7 = 3 \cdot 5 \cdot 7 = 105$, the solution to the congruences is that $x \equiv 62 \pmod{105}$.

</details>
